src/main/java/com/cleanroommc/groovyscript/helper/recipe/AbstractRecipeBuilder.java

```diff
@@ -162,12 +165,22 @@ public void validateFluids(GroovyLog.Msg msg, int minFluidInput, int maxFluidInp
         msg.add(fluidOutput.size() < minFluidOutput || fluidOutput.size() > maxFluidOutput, () -> getRequiredString(minFluidOutput, maxFluidOutput, "fluid output") + ", but found " + fluidOutput.size());
     }
 
+    @GroovyBlacklist
+    protected int getMaxInput() {
```

Comment:
would like to specify that this is for items, so `getMaxItemInput`. also, given that its `protected`, is there a need to have `@GroovyBlacklist` on it?

Reply:
I have no idea how GroovyBlacklist works ngl. Will rename to getMaxItemInput, sure.

Reply:
Or I mean like, what should or should not be blacklisted